README.md

```diff
@@ -9,9 +9,10 @@ This repo aggregates all audits performed on Sablier smart contracts by third-pa
 
 [5dc175c]: https://github.com/sablier-labs/flow/tree/5dc175cca189ba0401b1e877a62e5ca13a85384b
 
-| Auditors | Types | Commits              | Reports                                          |
-| :------- | :---- | :------------------- | :----------------------------------------------- |
-| Cantina  | Firm  | [`5dc175c`][5dc175c] | [2024-07-03](./lockup/v1.2/20240703_cantina.pdf) |
+| Auditors  | Types   | Commits              | Reports                                        |
+| :-------- | :------ | :------------------- | :--------------------------------------------- |
+| Cantina   | Firm    | [`5dc175c`][5dc175c] | [2024-10-20](./flow/v1.0/20241020_cantina.pdf) |
+| CodeHawks | Contest | [`5dc175c`][5dc175c] | [2024-11-13](./flow/v1.0/20241113_flow.md)     |
```

Comment:
The commit for Codehawk would be different i.e. post fix changes of Cantina findings.